### PR TITLE
upgrades_merge_faqfix

### PR DESCRIPTION
**Pull request by issa-me-sush**
#8141  - post  merge FAQ text fix - have updated with the relevant content for  the english version, should I also google translate for  the rest of the languages or  this is sufficient?
first contribution ever, hope its accepted :'D, thank you

### CODE REVIEW

**Comment by corwintines** [comment]
Will add back the .env.example file and combine these changes. Thanks @issa-me-sush.

File: src/intl/en/page-upgrades-index.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
   "page-upgrades-dive": "Dive into the vision",
   "page-upgrades-dive-desc": "How are we making Ethereum more scalable, secure, and sustainable? All while keeping Ethereum's core ethos of decentralization.",
   "page-upgrades-docking": "The Merge",
-  "page-upgrades-merge-answer-1": "The Merge was when Mainnet switched off proof-of-work and turned on proof-of-stake on 15th September 2022.",
+  "page-upgrades-merge-answer-1": "The merge was when Ethereum transitioned to proof-of-stake consensus on September 15, 2022. The Beacon Chain merged with Mainnet, officially deprecating proof-of-work on Ethereum, and reduced Ethereum’s energy consumption by ~99.95%.",
```
Comment (corwintines) [inline src/intl/en/page-upgrades-index.json:46]:
```suggestion
  "page-upgrades-merge-answer-1": "The Merge was when Ethereum transitioned to proof-of-stake consensus on September 15, 2022. The Beacon Chain merged with Mainnet, officially deprecating proof-of-work on Ethereum, and reduced Ethereum’s energy consumption by ~99.95%.",
```

**Review by corwintines — APPROVED** [review]
Thanks, @issa-me-sush!

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. We also have our 2022 POAPs so remember to [claim yours](https://ethereum.org/en/contributing/#poap) 🥳!

**Comment by corwintines** [comment]
@all-contributors can you add @issa-me-sush for doc

**Comment by issa-me-sush** [comment]
thank you so much, delighted that the changes were accepted :D !

**Comment by issa-me-sush** [comment]
already on the server and surely will try to contribute as much as possible in the coming days  :D

**Pull request merged**